#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#

uint8 volume
uint16 baseFrequency
uint16 cycleRate
uint8 dutyCycle
uint16 numberOfCycles
